shared/templates/accounts_password/bash.template

```diff
@@ -3,7 +3,6 @@
 # strategy = restrict
 # complexity = low
 # disruption = low
-. /usr/share/scap-security-guide/remediation_functions
 {{{ bash_instantiate_variables("var_password_pam_" + VARIABLE) }}}
```

Comment:
I have discovered that we can't remove the `. /usr/share/scap-security-guide/remediation_functions` line because there is the Jinja macro `bash_instantiate_variables` that gets expanded to a bash function `populate`. Please remove this removal. In other words, add it back.

This is a thing that could be revealed by SSGTS test suite. Unfortuantely, the GH action `SSGTS / Run Tests (pull_request)` has terminated unexpectedly, probably due to a problem that isn't related to your PR.